index.css

```diff
@@ -0,0 +1,166 @@
+/* stylelint-disable-next-line no-empty-first-line */
```

Comment:
- [ ] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `stylelint`, it will be nice to fix the issue.